### PR TITLE
#672/ Chain indexing support for total claimed rewards

### LINKED ISSUE #672: Problem: Support user claimed total rewards in CRO
- Get total rewards claimed by an account on CRO Mainnet
- https://crypto.org/explorer/api/v1/accounts/cro1gaf3jqqzvrxvgc4u4vr6x0tlf6kcm703zqa34a/messages?pagination=offset&page=1&memo=&limit=1&order=height.desc&filter.msgType=MsgWithdrawDelegatorReward

### PR DESCRIPTION
**Pull request by cdc-Hitesh**
Fixes #672

### CODE REVIEW

**Comment by crypto-3dd** [comment]
```
src/service/rpc/ChainIndexingAPI.ts
  Line 225:11:  'rewardMsgList' is never reassigned. Use 'const' instead  prefer-const
```

This might be failing the CI.

File: src/service/rpc/ChainIndexingAPI.ts [hunk 1]
```diff
@@ -213,4 +215,55 @@ export class ChainIndexingAPI implements IChainIndexingAPI {
...
+    }
+  }
+
+  private async getDelegatorRewardMessageList(address: string) {
+
+    // Note: handle `pagination`
+    const delegatorRewardMessageList = await this.axiosClient.get<AccountMessagesListResponse>(
+      `accounts/${address}/messages?order=height.desc&filter.msgType=MsgWithdrawDelegatorReward`,
+    );
+
```
Comment (crypto-3dd) [inline src/service/rpc/ChainIndexingAPI.ts:260]: I think we would just go ahead and paginate it directly.

Comment (cdc-Hitesh) [reply src/service/rpc/ChainIndexingAPI.ts:260]: Do you mean, 
We should handle it now?
or 
This will be handled automatically?

@crypto-3dd

Comment (crypto-3dd) [reply src/service/rpc/ChainIndexingAPI.ts:260]: > We should handle it now?

Yes, I mean handling it now.

Comment (cdc-Hitesh) [reply src/service/rpc/ChainIndexingAPI.ts]: I have handled pagination now.

File: src/service/rpc/ChainIndexingAPI.ts [hunk 2]
```diff
@@ -20,6 +22,7 @@ import { DefaultWalletConfigs } from '../../config/StaticConfig';
 import { croNftApi, MintByCDCRequest } from './NftApi';
 import { splitToChunks } from '../../utils/utils';
 import { UserAsset } from '../../models/UserAsset';
+import { accountMsgList } from './ChainIndexingModels';
```
Comment (crypto-3dd) [inline src/service/rpc/ChainIndexingAPI.ts:25]: `'./ChainIndexingModels';` is being imported multiple times. This makes the CI fail

**Review by crypto-3dd — DISMISSED** [review]

File: src/service/rpc/ChainIndexingAPI.ts (hunk 2 above)
Comment (cdc-Hitesh) [reply src/service/rpc/ChainIndexingAPI.ts]: Fixed and thanks for the tip Eddy.

My eslint seems to not fix this 😢 , not sure what's wrong

**Review by crypto-3dd — APPROVED** [review]

**Pull request merged**